lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1427,6 +1427,45 @@ class CreateThreadTest(
         SharedModuleStoreTestCase,
         MockSignalHandlerMixin
 ):
+    LONG_TITLE = (
```

Comment:
I agree, using a specific text makes this test much better than if it were a generated string